settings_constant.py

```diff
@@ -26,4 +26,9 @@
 TOKEN_KEY = "transition_qgis/token"
 URL_KEY = "transition_qgis/url"
 USERNAME_KEY = "transition_qgis/username"
-KEEP_CONNECTION_KEY = "transition_qgis/keepConnection"
\ No newline at end of file
+KEEP_CONNECTION_KEY = "transition_qgis/keepConnection"
+
+# Function to convert a string to a boolean. Settings values are kept as string
+# in the files, so need to be converted to boolean if necessary
+def settingsValueToBool(value):
```

Comment:
Are you sure that QGIS don't provide this as a function somewhere? Kinda strange to not have it already